mitodl/mitxpro
# Add modal selection widget for enrollment code purchase form

#### Pre-Flight checklist


#### What are the relevant tickets?
Part of #858 

#### What's this PR do?
Replaces the standard HTML select with a modal dropdown element

#### How should this be manually tested?
Go to `/ecommerce/bulk/` and select some courses and programs. It should look similar to the mockup in the linked issue. Please comment if the UI feels strange or if you notice anything which should be changed.

#### Screenshots (if appropriate)
Desktop:
![Screenshot from 2019-08-20 13-47-49](https://user-images.githubusercontent.com/863262/63371082-64a7da00-c351-11e9-8eae-705e62ab076d.png)

Mobile:
![Screenshot from 2019-08-20 13-48-12](https://user-images.githubusercontent.com/863262/63371089-68d3f780-c351-11e9-8df8-5abdab58bb03.png)
